vortex-array/src/array/bool/stats.rs

```diff
@@ -43,11 +43,12 @@ impl StatisticsVTable<NullableBools<'_>> for BoolEncoding {
         // Fast-path if we just want the true-count
         if matches!(
             stat,
-            Stat::TrueCount | Stat::Min | Stat::Max | Stat::IsConstant
+            Stat::TrueCount | Stat::Min | Stat::Max | Stat::IsConstant | Stat::NullCount
         ) {
+            let _null_count = array.1.count_set_bits();
             return Ok(StatsSet::bools_with_true_and_null_count(
                 array.0.bitand(array.1).count_set_bits(),
-                array.1.count_set_bits(),
+                array.1.len() - array.1.count_set_bits(),
```

Comment:
🤦 luckily we weren't using this value before because we fall back to the accumulator for null count. But null count != valid count.

Reply:
Eh, this is the valid count not null count. Apologies